--- noms_display.cfg	2020-05-31 11:06:39.295685700 -0400
+++ display.cfg	2020-05-31 11:06:18.411405000 -0400
@@ -55,18 +55,13 @@
     { "%6s" % (msg,) }
   {% endif %}
 
-[display_template _toolhead_pos]
-text:
-  {% set pos = printer.toolhead.position %}
-  { "X%-4.0fY%-4.0fZ%-5.2f" % (pos.x, pos.y, pos.z) }
-
 [display_template _print_status]
 text:
   {% if printer.display_status.message %}
     { printer.display_status.message }
-  #{% elif printer.idle_timeout.printing_time or printer.gcode.busy %}
-  #  {% set pos = printer.toolhead.position %}
-  #  { "X%-4.0fY%-4.0fZ%-5.2f" % (pos.x, pos.y, pos.z) }
+  {% elif printer.idle_timeout.printing_time or printer.gcode.busy %}
+    {% set pos = printer.toolhead.position %}
+    { "X%-4.0fY%-4.0fZ%-5.2f" % (pos.x, pos.y, pos.z) }
   {% else %}
     Ready
   {% endif %}
@@ -97,18 +92,15 @@
   { "{:>4.0%}".format(printer.gcode.speed_factor) }
 
 [display_data _default_16x4 print_progress]
-position: 2,0
-text: { render("_toolhead_pos") }
-#position: 2, 0
-#text: { "{:^10.0%}".format(printer.display_status.progress) }
-#[display_data _default_16x4 progress_bar]
-#position: 2, 1 # Draw graphical progress bar after text is written
-#text: { draw_progress_bar(2, 0, 10, printer.display_status.progress) }
-#
-#[display_data _default_16x4 printing_time]
-#position: 2, 10
-#text: { render("_printing_time") }
-
+position: 2, 0
+text: { "{:^10.0%}".format(printer.display_status.progress) }
+[display_data _default_16x4 progress_bar]
+position: 2, 1 # Draw graphical progress bar after text is written
+text: { draw_progress_bar(2, 0, 10, printer.display_status.progress) }
+
+[display_data _default_16x4 printing_time]
+position: 2, 10
+text: { render("_printing_time") }
 
 [display_data _default_16x4 print_status]
 position: 3, 0
